### ISSUE TITLE
Code for reproduce imagenet training

### ISSUE THREAD

**Issue by kamikawa** [keras-team/keras#8536]
Hi,

Thank you for a great deep learning frame work for newbie.
But I found it difficult for me to reproduce imagenet training result of paper by such as resnet,
which can be done by other frame works.
Could you open a code for reproduce imagenet training whole code for such as Xception model ?

I guess what is missing is;
- weight decay setting (kernel regularizer) 
- initializer setting 
- SGD scheduler callback function (most SOTA results are acquired by SGD)
- preprocessing part 

Best regards.
